gcp/workers/linter/linter.py

```diff
@@ -147,6 +149,38 @@ def record_quality_finding(bug_id: str, source: str,
                   new_findings, source)
 
 
+def upload_record_to_bucket(json_result: Any):
+  """Uploads the linter result to a GCS bucket, creating one file per source."""
+  storage_client = storage.Client()
+  bucket = storage_client.get_bucket(LINTER_EXPORT_BUCKET)
+  prefix_to_source = construct_prefix_to_source_map()
```

Comment:
Out of scope of this PR, but I do wonder why there is this construct function and why it's not just a global variable.